src/LinAlg/hiopMatrixDenseRowMajor.cpp

```diff
@@ -237,15 +237,15 @@ void hiopMatrixDenseRowMajor::copyFromMatrixBlock(const hiopMatrixDense& srcmat,
 void hiopMatrixDenseRowMajor::shiftRows(size_type shift)
 {
   if(shift==0) return;
-  if(fabs(shift)==m_local_) return; //nothing to shift
+  if(std::fabs(shift)==m_local_) return; //nothing to shift
```

Comment:
Why is `fabs()` used here? Did you intend to use `std::abs()`? There are several other places in the code where `fabs` is used with integer arguments and where the return value is also expected to be an integer. Building in developer mode or with `-Wconversion` flag should catch these instances.

Reply:
I kept the function the same and just added the `std` namespace qualifier, but it does seem like a few of these should have originally been `abs` instead of `fabs`. I can changes these.